.github/workflows/test.yml

```
@@ -76,7 +76,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu]
-        python-version: ["3.9", "3.10"]
+        python-version: ["3.10", "3.11", "3.12"]
```

And since this is the "prerelease" job it's probably fine to keep it to the two newest ones I think
```suggestion
        python-version: ["3.11", "3.12"]
```